<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioClone</class>
 <widget class="QWidget" name="AudioClone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音频克隆</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>音频克隆</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 24px;
font-weight: bold;
color: #2c3e50;
padding: 10px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="referenceGroup">
     <property name="title">
      <string>参考音频</string>
     </property>
     <layout class="QVBoxLayout" name="referenceLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>选择参考音频文件：</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="refAudioLayout">
        <item>
         <widget class="QLineEdit" name="refAudioPath">
          <property name="placeholderText">
           <string>未选择文件</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectRefAudio">
          <property name="text">
           <string>浏览...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="textGroup">
     <property name="title">
      <string>合成文本</string>
     </property>
     <layout class="QVBoxLayout" name="textLayout">
      <item>
       <widget class="QTextEdit" name="textInput">
        <property name="placeholderText">
         <string>请输入要合成的文本内容...</string>
        </property>
        <property name="minimumHeight">
         <number>150</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="optionsGroup">
     <property name="title">
      <string>选项</string>
     </property>
     <layout class="QVBoxLayout" name="optionsLayout">
      <item>
       <layout class="QHBoxLayout" name="pitchLayout">
        <item>
         <widget class="QLabel" name="pitchLabel">
          <property name="text">
           <string>音调调整：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="pitchSlider">
          <property name="minimum">
           <number>-12</number>
          </property>
          <property name="maximum">
           <number>12</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pitchValue">
          <property name="text">
           <string>0</string>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="btnGenerate">
       <property name="text">
        <string>生成音频</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     background-color: #27ae60;
     color: white;
     border-radius: 5px;
     font-size: 14px;
     padding: 8px 16px;
 }

 QPushButton:hover {
     background-color: #229954;
 }

 QPushButton:disabled {
     background-color: #bdc3c7;
 }</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPlay">
       <property name="text">
        <string>播放</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>保存</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>